src/css/home.scss

```diff
@@ -242,6 +242,109 @@
 
     }
 
+    .exchange-list {
+        margin-bottom: 3.12rem;
+        background-color: #{$black};
+
+        .exchange-list-container {
```

Comment:
Maybe let's add some `margin-bottom` to the `exchange-list-container`. Thoughts?